### DISCUSSION TITLE
apache-airflow-providers-apache-hive support missing from constraints file for Python 3.11 Airflow 2.7.x

### DISCUSSION THREAD

**Discussion by andscoop** [apache/airflow#36099] [General]
### Apache Airflow version

2.7.3

### What happened

`apache-airflow-providers-apache-hive` is missing from the contraints file(s) for Python 3.11. Reviewing some of the context, it looks like [hive support was re-added in Airflow 2.7.1,](https://github.com/apache/airflow/pull/32607) but I am not seeing it in the [latest constraint file for python 3.11 / Airflow 2.7.3](https://raw.githubusercontent.com/apache/airflow/constraints-2.7.3/constraints-3.11.txt).

### How to reproduce

N/A

### Operating System

N/A

### Versions of Apache Airflow Providers

apache-airflow-providers-apache-hive

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
Thanks for reporting - yes. it's been removed from generating constraints (and only recently it's been added back). It's already there in https://github.com/apache/airflow/blob/constraints-2.8.0b1/constraints-3.11.txt 

No big problem with it - no big harm in single provider missing. Not a reason to update the constraints, unless you have  good reason of course.

Converted it into discussion if more needed.

**Comment by andscoop** [comment]
My understanding of the constraints is that it shows which versions of dependencies are proven to be compatible with that airflow / python version combo. I'm just curious if it is missing because it is not compatible with 2.7.3/ 3.11 or it was left out by mistake.

**Reply by potiuk** [reply]
Yeah mistake. The code excluding give for python 3.11 exclusion for constraint generation had been removed only recently. I might actually want to fix it anyway as it is a base for our SBOM generation

**Reply by potiuk** [reply]
OK. 2.7.1, 2.7.2 and 2.7.3 had been updated (see for example here: https://github.com/apache/airflow/blob/constraints-2.7.2/constraints-3.11.txt) 

Thanks for being vigilant!

**Reply by andscoop** [reply]
Thanks for the quick answer and fix. 🙇
